HEC.FDA.Statistics/Distributions/IDistributionFactory.cs

```diff
@@ -38,13 +38,19 @@ public static string PrintParamaterizationRequirements(IDistributionEnum type)
                     throw new NotImplementedException();
             }
         }
-
+        /// <summary>
+        /// we have to keep this because of IDistribution
+        /// </summary>
+        /// <param name="sample"></param>
+        /// <param name="nBins"></param>
+        /// <returns></returns>
         internal static Statistics.Histograms.Histogram Fit(IEnumerable<double> sample, int nBins)
```

Comment:
IDistribution has a bunch of baggage. That interface needs broken up into multiple interfaces so classes don't have to implement bogus methods.

Reply:
Also, Delete IDistribution Factory and replace it with the proper constructors all over the place. It's only used in tests .